docs/hub/spaces-config-reference.md

```diff
@@ -32,7 +32,10 @@ All versions of Streamlit from `0.79.0` are supported.
 Specify the suggested [hardware](https://huggingface.co/docs/hub/spaces-gpus) on which this Space must be run.  
 Useful for Spaces that are meant to be duplicated by other users.  
 Setting this value will not automatically assign an hardware to this Space.  
-Value must be a valid hardware flavor (e.g. `"cpu-upgrade"`, `"t4-small"`, `"t4-medium"`, `"a10g-small"`, `"a10g-large"`, `"a10g-largex2"`, `"a10g-largex4"` or `"a100-large"`).  
+Value must be a valid hardware flavor (e.g. `"cpu-upgrade"`, `"t4-small"`, `"t4-medium"`, `"l4x1"`,
+	`"l4x4"`, `"a10g-small"`, `"a10g-large"`, `"a10g-largex2"`,
+	`"a10g-largex4"`,`"a100-large"`, `"v5e-1x1"`,
+	`"v5e-2x2"`, `"v5e-2x4"`).  
```

Comment:
```suggestion
Value must be a valid hardware flavor. Current valid hardware flavors:
- CPU: `"cpu-basic"`, `"cpu-upgrade"`
- GPU: `"t4-small"`, `"t4-medium"`, `"l4x1"`,
	`"l4x4"`, `"a10g-small"`, `"a10g-large"`, `"a10g-largex2"`,
	`"a10g-largex4"`,`"a100-large"`
- TPU: `"v5e-1x1"`, `"v5e-2x2"`, `"v5e-2x4"`
```
@julien-c I agree it is not clear that those are TPUs, here's an alternative formatting suggestion for this part of the docs

Reply:
yes i agree that's better! 

btw @christophe-rannou maybe we could have encoded tpu inside the strings `"tpu-v5e-1x1"` etc

Reply:
Yes this is nicer